### PR TITLE
Alpha content review

### PR DESCRIPTION
**Pull request by bradmorton1**
## Summary
updated the API spec

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: specification/nhs-website-content-api.yaml [hunk 1]
```diff
@@ -6,21 +6,20 @@ info:
...
+
     You can reuse the following content from the NHS website:
-      - health conditions
-      - medicines
-      - pregnancy
-      - mental health
-      - videos
-      - common health questions
-      - information about NHS services
+      * health conditions
```
Comment (adrianparr) [inline specification/nhs-website-content-api.yaml:12]: Should 'live well' be listed in here too?

File: specification/nhs-website-content-api.yaml [hunk 2]
```diff
@@ -32,11 +31,19 @@ info:
       TODO link to API docs
 
     ## API status and roadmap
-    This API has been moved from the nhs.uk developer API website. It is currently in Alpha phase.
+    This API has been moved from the nhs.uk developer API website. It is currently in Alpha phase. If you are an existing user, you need to sign up for the API in the new portal.
```
Comment (adrianparr) [inline specification/nhs-website-content-api.yaml:33]: Can this sentence say ...
'If you are an existing user, you need to sign up for the API in the new API Platform.'

File: specification/nhs-website-content-api.yaml [hunk 3]
```diff
@@ -52,16 +59,41 @@ info:
...
     ## Network access
-    No special network access is required for this API.
+    No special network access is required for this API. It is available over the internet.
 
     ## Security and authorisation
-    You will need to provide a subscription key in the header or as a query parameter in your requests.
+    This API is [application-restricted](https://digital.nhs.uk/developer/guides-and-documentation/security-and-authorisation#application-restricted-apis), meaning we authenticate the calling application but not the end user.
 
-    Your subscription key can be found by logging into the API portal (provide link)
+    The end user will need to provide a subscription key in the header or as a query parameter in requests.
```
Comment (adrianparr) [inline specification/nhs-website-content-api.yaml:66]: In the new API Platform 'a subscription key' are referred to as 'an API key'.
Maybe we should update this to match.

File: specification/nhs-website-content-api.yaml [hunk 4]
```diff
@@ -52,16 +59,41 @@ info:
...
+
+    Integration testing
+    Our integration test environment:
+
+    * is for formal integration testing.
+    * includes application authentication
+    * contains a replica of our live data, but has a reduced call rate
+
     ## Onboarding
+    You need to sign up via the portal to recieve your subscription key.
```
Comment (adrianparr) [inline specification/nhs-website-content-api.yaml:82]: 'portal'   --->    'API Platform'

File: specification/nhs-website-content-api.yaml [hunk 5]
```diff
@@ -70,15 +102,296 @@ info:
 
     The logo should be clearly visible and in line with the associated content on every web page that contains the syndicated content. You should also link this logo to the relevant page on the NHS website that the content has been supplied from. This URL is supplied as part of the feed under author:
 
+    ## Errors
+
+    We have standard error codes for 200 to 299 if it succeeded, including code 202 if it was accepted by an API that needs to wait for further action
+
+    Currently, our akamai cache provides other error codes in the form of a webpage.
```
Comment (adrianparr) [inline specification/nhs-website-content-api.yaml:108]: Akamai

File: specification/nhs-website-content-api.yaml [hunk 6]
```diff
@@ -70,15 +102,296 @@ info:
...
+paths:
+  /conditions:
+    parameters:
+      - $ref: "#/components/parameters/subscription-key"
+    get:
+      summary: Get all pages in the [health A-Z](https://www.nhs.uk/conditions)
+      operationId: get-conditions
+      description: |
+        ## Overview
+        Use this endpoint to get details of pages in the health A-Z.
```
Comment (adrianparr) [inline specification/nhs-website-content-api.yaml:131]: Health A to Z

File: specification/nhs-website-content-api.yaml [hunk 7]
```diff
@@ -70,15 +102,296 @@ info:
...
+              example:
+                $ref: components/examples/Conditions.json
+  /conditions/*:
+    parameters:
+      - $ref: "#/components/parameters/subscription-key"
+    get:
+      summary: Get details of condition page
+      operationId: get-condition-page
+      description: |
+        There is an endpoint for each page under the health A-Z e.g. conditions/chickenpox
```
Comment (adrianparr) [inline specification/nhs-website-content-api.yaml:179]: Health A to Z

File: specification/nhs-website-content-api.yaml [hunk 8]
```diff
@@ -70,15 +102,296 @@ info:
...
+          application/schema+json:
+            schema:
+              $ref: "#/components/schemas/ConditionPage"
+            example:
+              $ref: components/examples/ConditionPage.json
+  /medicines:
+    parameters:
+      - $ref: "#/components/parameters/subscription-key"
+    get:
+      summary: Get all pages in the [medicines A-Z](https://www.nhs.uk/medicines)
```
Comment (adrianparr) [inline specification/nhs-website-content-api.yaml:212]: Medicines A to Z

File: specification/nhs-website-content-api.yaml [hunk 9]
```diff
@@ -70,15 +102,296 @@ info:
...
+      operationId: get-medicines
+      description: |
+        ## Overview
+        Use this endpoint to get details of pages in the medicines A-Z.
```
Comment (adrianparr) [inline specification/nhs-website-content-api.yaml:216]: Medicines A to Z

File: specification/nhs-website-content-api.yaml [hunk 10]
```diff
@@ -70,15 +102,296 @@ info:
...
+              example:
+                $ref: components/examples/Medicines.json
+  /medicines/*:
+    parameters:
+      - $ref: "#/components/parameters/subscription-key"
+    get:
+      summary: Get specific medicines pages
+      operationId: get-medicine-page
+      description: |
+        There is an endpoint for each page under the medicines A-Z e.g. /medicines/amoxicillin
```
Comment (adrianparr) [inline specification/nhs-website-content-api.yaml:267]: Medicines A to Z

File: specification/nhs-website-content-api.yaml [hunk 11]
```diff
@@ -70,15 +102,296 @@ info:
...
+            application/schema+json:
+              schema:
+                $ref: "#/components/schemas/CommonHealthQuestions"
+              example:
+                $ref: components/examples/CommonHealthQuestions.json
+  /live-well:
+    parameters:
+      - $ref: "#/components/parameters/subscription-key"
+    get:
+      summary: Get content from the live-well hub.
```
Comment (adrianparr) [inline specification/nhs-website-content-api.yaml:373]: Live Well hub

File: specification/nhs-website-content-api.yaml [hunk 12]
```diff
@@ -70,15 +102,296 @@ info:
...
+      operationId: get-common-health-questions
+      description: |
+        The live-well hub contains a number of navigational and content pages.
```
Comment (adrianparr) [inline specification/nhs-website-content-api.yaml:376]: Live Well hub

File: specification/nhs-website-content-api.yaml [hunk 13]
```diff
@@ -70,15 +102,296 @@ info:
...
+
+        The content of the page can be taken from mainEntityOfPage. This includes the navigational patterns to child pages
+
+        At the top level, child modularised content is highlighted by the hasPart field. In child pages, only the individual page will be highlighted
+
+      responses:
+        "200":
+          description: |
+            A valid query for the live-well pages.
```
Comment (adrianparr) [inline specification/nhs-website-content-api.yaml:385]: Live Well pages

File: specification/nhs-website-content-api.yaml (hunk 1 above)
Comment (bradmorton1) [reply specification/nhs-website-content-api.yaml:12]: Yes and not videos

**Review by adrianparr — APPROVED** [review]
Approved

**Pull request merged**